src/unicode/unicode.cr

```diff
@@ -1,13 +1,7 @@
-# Provides methods that answer questions about unicode characters,
-# and the `Unicode::CaseOptions` enum for special case conversions
-# like Turkic.
-#
-# There's no need to use the methods defined in this module
-# because they are exposed in `Char` in a more convenient way
-# (`Char#upcase`, `Char#downcase`, `Char#whitespace?`, etc.)
+# Provides the `Unicode::CaseOptions` enum for special case conversions like Turkic and
+# private and undocumented methods that are exposed in `Char`.
```

Comment:
I'd rather not reference the undocumented methods at all. Leave this class with no doc comment and seperate the old doc comment from the module with a line.

Reply:
You mean like this?
```cr
# ...
# because they are exposed in `Char` in a more convenient way
# (`Char#upcase`, `Char#downcase`, `Char#whitespace?`, etc.)

# :nodoc:
module Unicode
...
```
But then the module wouldn't show up in the API documentation and so neither the `CaseOptions` enum.

Reply:
"no doc comment" != "nodoc comment"

In your example, simply remove the nodoc line. `Unicode` will be included in the docs, but the documentation for this module will be empty.